translate 2020 Capabilities chapter to Dutch (#1889)
HTTPArchive/almanac.httparchive.org: src/content/nl/2020/capabilities.md

---
part_number: II
chapter_number: 13
title: Mogelijkheden
description: Het hoofdstuk Mogelijkheden van de Web Almanac 2020 met betrekking tot gloednieuwe, krachtige webplatform-API's.
authors: [christianliebel]
reviewers: [tomayac]

> analysts: [tomayac]

Reviewer: We added editors recently:

```suggestion
analysts: [tomayac]
editors: [rviscomi]
```

### Schrijftoegang

Afgezien van leesbewerkingen biedt de Async Clipboard API ook twee methoden om inhoud naar het klembord te schrijven. Nogmaals, er is een verkorte methode voor platte tekst, genaamd `navigator.clipboard.writeText()`, en een voor willekeurige gegevens genaamd `navigator.clipboard.write()`. In op Chromium gebaseerde browsers is geen toestemming vereist om naar het klembord te schrijven terwijl het tabblad actief is. Proberen naar het klembord te schrijven wanneer de website op de achtergrond staat, is wel toestemming voor nodig. Aangezien deze methode eerst een gebruikersgebaar en toestemming vereist, valt deze niet onder de HTTP Archive-statistieken. In tegenstelling tot de `read()` - methode, laat de `write()` - methode een exponentiële groei in gebruik zien, die deel uitmaakt van 0,0006% van alle paginaladingen in oktober 2020.

De <a hreflang="en" href="https://bugs.chromium.org/p/chromium/issues/detail?id=897289">Raw Clipboard Access API</a>, een andere Fugu-mogelijkheid, zou de Async Clipboard API nog verder kunnen verbeteren door het mogelijk te maken dat willekeurige gegevens worden gekopieerd van of geplakt op het klembord.

> ## StorageManager API

Reviewer: We've added `<span lang="en">` to some of these headings but not others. Any reason or just missed some?

I think it's arguable whether they are needed or not so happy either way, but should be consistent and either always use it or not.

  image="storage_manager_api_persist.png",
  alt="Aantal pagina's dat de `persist`-methode van de StorageManager API gebruikt.",
  caption="Aantal pagina's dat de `persist`-methode van de StorageManager API gebruikt.",
  description="Grafiek van het gebruik van de `persist`-methode van StorageManager API, op basis van het aantal pagina's dat wordt gecontroleerd door HTTP Archive. Het vergelijkt het gebruik op mobiele en desktop-apparaten. Op desktoppagina's is het gebruik nagenoeg stabiel, terwijl er meer fluctuatie is op mobiele apparaten. In oktober 2020 maken 25 desktoppagina's en 176 mobiele pagina's gebruik van de API.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=644836316&format=interactive",
  sheets_gid="1095648844",
  sql_file="durable_storage_persist_usage.sql"
  )
}}

De `persist()` API wordt minder vaak aangeroepen dan de `estimate()` methode. Slechts 176 mobiele pagina's maken gebruik van deze API, vergeleken met 25 desktopwebsites. Hoewel het gebruik op de desktop op dit lage niveau lijkt te blijven, is er geen duidelijke trend op mobiele apparaten.

> ## New Notification APIs

Reviewer: Should we translate New? What about Notifications?

```suggestion
## Nieuw Notification APIs
```

Met behulp van de Push- en Notifications-API's kunnen webapplicaties al lang pushberichten ontvangen en meldingsbanners weergeven. Er ontbraken echter enkele onderdelen. Tot nu toe moesten push-berichten via de server worden verzonden; ze konden niet offline worden gepland. Ook konden webtoepassingen die op het systeem waren geïnstalleerd, geen badges op hun pictogram weergeven. De API's voor badging en Notification Triggers maken beide scenario's mogelijk.

> ### Badging API

Reviewer: To span or not to span?

  description="Grafiek van Badging API-gebruik, gebaseerd op het percentage pagina's dat met deze functie in Chrome wordt geladen. Het vergelijkt de `set` en `clear` methoden. Het gebruik van beide methoden groeit in de loop van de tijd, waarbij de `set` methode over het algemeen vaker wordt aangeroepen. In oktober 2020 is er een plotselinge groei voor beide methoden, met een piek van 0,025% van het laden van pagina's voor de `set` methode en 0,016% voor `clear`.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1145004925&format=interactive",
  sheets_gid="1154751352"
  )
}}

In april 2020 heeft Google Chrome 81 de nieuwe Badging API verzonden, gevolgd door Microsoft Edge 84 in juli. Nadat Chrome de API had verzonden, schoten de gebruikscijfers omhoog. In oktober 2020 wordt bij 0,025% van alle pagina's die in Google Chrome worden geladen, de `setAppBadge()` - methode aangeroepen. De `clearAppBadge()` - methode wordt minder vaak aangeroepen, wanneer ongeveer 0,016% van de pagina's wordt geladen.

> ### Notification Triggers API

Reviewer: Again should this be spanned?

  description="Diagram met Notification Triggers API-gebruik, gebaseerd op het percentage pagina's dat in Chrome wordt geladen met deze functie. Het vertoont een piek in maart 2020 met ongeveer 0,00003% van de paginaladingen, en daalt tot nul in oktober 2020.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1388597384&format=interactive",
  sheets_gid="1740370570"
  )
}}

Het Fugu-team experimenteerde eerst met Notification Triggers in een oorspronkelijke proef van Chrome 80 tot 83, waarbij de ontwikkeling later werd onderbroken vanwege het gebrek aan feedback van ontwikkelaars. Beginnend met Chrome 86 uitgebracht in oktober 2020, is de API weer in de oorspronkelijke proeffase terechtgekomen. Dit verklaart ook de gebruiksgegevens van de Notification Triggers API die piekten toen ze werden aangeroepen op 0,000032% van de pagina's die in Chrome werden geladen tijdens de eerste oorspronkelijke proef rond maart 2020.

> ## Screen Wake Lock API

Reviewer: And here.

  console.log(app.id, app.platform, app.url);
});
```

{{ figure_markup(
  image="get_installed_related_apps.png",
  caption="Aantal pagina's dat `getInstalledRelatedApps()` gebruikt.",
  description="Grafiek van het gebruik van `getInstalledRelatedApps()`, op basis van het aantal pagina's dat wordt gecontroleerd door HTTP Archive. Het vergelijkt het gebruik op mobiele en desktop-apparaten. Het laat een gestage groei zien voor mobiele apparaten, met een piek van 363 pagina's in oktober 2020 vergeleken met 44 desktoppagina's.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1774881171&format=interactive",
  sheets_gid="860146688",
  sql_file="get_installed_related_apps_usage.sql"
  )
}}

In de loop van 2020 vertoont de `getInstalledRelatedApps()` API een gestage groei op mobiele websites. In oktober maakten 363 mobiele pagina's die werden gevolgd door het HTTP Archive gebruik van deze API. Op desktoppagina's groeit de API niet zo snel. Dit kan ook het gevolg zijn van het feit dat Android-winkels momenteel aanzienlijk meer apps aanbieden dan de Microsoft Store voor Windows.

> ## Content Indexing API

Reviewer: And here.

De Content Indexing API breidt de Service Worker API uit met een nieuwe `ContentIndex` interface. Deze interface is beschikbaar via de eigenschap `index` van de registratie van de Service Worker. Met de `add()` methode kunnen ontwikkelaars inhoud aan de index toevoegen: elk stukje inhoud moet een ID, een URL, een start-URL, titel, beschrijving en een reeks pictogrammen hebben. Optioneel kan de inhoud worden gegroepeerd in verschillende categorieën, zoals artikelen, homepagina's of video's. De `delete()` methode maakt het mogelijk om weer inhoud uit de index te verwijderen, en de `getAll()` methode geeft een lijst met alle geïndexeerde items terug.

{{ figure_markup(
  image="content_indexing_api.png",
  alt="Percentage pagina's dat wordt geladen in Chrome met behulp van Content Indexing API",

>   caption='Percentage pagina's dat wordt geladen in Chrome met behulp van Content Indexing API.<br>(Bron: <a hreflang="en" href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">Content Indexing</a>)',

Reviewer: ```suggestion
  caption='Percentage pagina\'s dat wordt geladen in Chrome met behulp van Content Indexing API.<br>(Bron: <a hreflang="en" href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">Content Indexing</a>)',
```

  description="Grafiek van het gebruik van Content Indexing API, gebaseerd op het percentage pagina's dat met deze functie in Chrome wordt geladen. Het vertoont in eerste instantie een relatief laag gebruik, totdat het in oktober 2020 plotseling vertienvoudigt, en wordt gebruikt tijdens 0,0021% van de paginaladingen in Chrome.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=258329620&format=interactive",
  sheets_gid="626752011"
  )
}}

De Content Indexing API is gelanceerd met Chrome 84 in juli 2020. Direct na verzending werd de API gebruikt tijdens ongeveer 0,0002% van de paginaladingen in Chrome. In oktober 2020 is deze waarde bijna vertienvoudigd.

> ## New Transport APIs

Reviewer: I think this should be translated as this is not actually an API name

```suggestion
## Nieuwe transport-API's
```

```js
const wss = new WebSocketStream(WSS_URL);
const {readable, writable} = await wss.connection;
const reader = readable.getReader();
const writer = writable.getWriter();
```

De WebSocketStream API lost op transparante wijze tegendruk op, aangezien de streamlezers en -schrijvers alleen doorgaan als het veilig is om dit te doen.

{{ figure_markup(
  image="websocketstreams.png",
  alt="Percentage pagina's dat in Chrome wordt geladen met WebSocketStreams",

>   caption='Percentage pagina's dat in Chrome wordt geladen met WebSocketStreams.<br>(Bron: <a hreflang="en" href="https://chromestatus.com/metrics/feature/timeline/popularity/3018">WebSocketStream</a>)',

Reviewer: ```suggestion
  caption='Percentage pagina\'s dat in Chrome wordt geladen met WebSocketStreams.<br>(Bron: <a hreflang="en" href="https://chromestatus.com/metrics/feature/timeline/popularity/3018">WebSocketStream</a>)',
```

```js
const transport = new QuicTransport(QUIC_URL);
await transport.ready;

const ws = transport.sendDatagrams();
const stream = await transport.createSendStream();
```

QuicTransport is een geldig alternatief voor WebSockets, omdat het de gebruiksscenario's van de WebSocket API ondersteunt en ondersteuning toevoegt voor scenario's waarin minimale latentie belangrijker is dan betrouwbaarheid en berichtvolgorde. Dit maakt het een goede keuze voor games en apps die te maken hebben met hoogfrequente gebeurtenissen.

{{ figure_markup(
  image="quic_transport.png",
  alt="Percentage pagina's dat in Chrome wordt geladen met Quic Transport",

>   caption='Percentage pagina's dat in Chrome wordt geladen met Quic Transport.<br>(Bron: <a hreflang="en" href="https://chromestatus.com/metrics/feature/timeline/popularity/3184">QuicTransport</a>)',

Reviewer: ```suggestion
  caption='Percentage pagina\'s dat in Chrome wordt geladen met Quic Transport.<br>(Bron: <a hreflang="en" href="https://chromestatus.com/metrics/feature/timeline/popularity/3184">QuicTransport</a>)',
```

### Schrijftoegang

Afgezien van leesbewerkingen biedt de Async Clipboard API ook twee methoden om inhoud naar het klembord te schrijven. Nogmaals, er is een verkorte methode voor platte tekst, genaamd `navigator.clipboard.writeText()`, en een voor willekeurige gegevens genaamd `navigator.clipboard.write()`. In op Chromium gebaseerde browsers is geen toestemming vereist om naar het klembord te schrijven terwijl het tabblad actief is. Proberen naar het klembord te schrijven wanneer de website op de achtergrond staat, is wel toestemming voor nodig. Aangezien deze methode eerst een gebruikersgebaar en toestemming vereist, valt deze niet onder de HTTP Archive-statistieken. In tegenstelling tot de `read()` - methode, laat de `write()` - methode een exponentiële groei in gebruik zien, die deel uitmaakt van 0,0006% van alle paginaladingen in oktober 2020.

De <a hreflang="en" href="https://bugs.chromium.org/p/chromium/issues/detail?id=897289">Raw Clipboard Access API</a>, een andere Fugu-mogelijkheid, zou de Async Clipboard API nog verder kunnen verbeteren door het mogelijk te maken dat willekeurige gegevens worden gekopieerd van of geplakt op het klembord.

> ## <span lang="en">StorageManager API</span>

Reviewer: ```suggestion
## <span lang="en">StorageManager API</span> {storagemanager-api}
```

  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=644836316&format=interactive",
  sheets_gid="1095648844",
  sql_file="durable_storage_persist_usage.sql"
  )
}}

De `persist()` API wordt minder vaak aangeroepen dan de `estimate()` methode. Slechts 176 mobiele pagina's maken gebruik van deze API, vergeleken met 25 desktopwebsites. Hoewel het gebruik op de desktop op dit lage niveau lijkt te blijven, is er geen duidelijke trend op mobiele apparaten.

## Nieuwe Notificatie APIs

Met behulp van de Push- en Notifications-API's kunnen webapplicaties al lang pushberichten ontvangen en meldingsbanners weergeven. Er ontbraken echter enkele onderdelen. Tot nu toe moesten push-berichten via de server worden verzonden; ze konden niet offline worden gepland. Ook konden webtoepassingen die op het systeem waren geïnstalleerd, geen badges op hun pictogram weergeven. De API's voor badging en Notification Triggers maken beide scenario's mogelijk.

> ### <span lang="en">Badging API</span>

Reviewer: ```suggestion
### <span lang="en">Badging API</span> {badging-api}
```

  description="Grafiek van Badging API-gebruik, gebaseerd op het percentage pagina's dat met deze functie in Chrome wordt geladen. Het vergelijkt de `set` en `clear` methoden. Het gebruik van beide methoden groeit in de loop van de tijd, waarbij de `set` methode over het algemeen vaker wordt aangeroepen. In oktober 2020 is er een plotselinge groei voor beide methoden, met een piek van 0,025% van het laden van pagina's voor de `set` methode en 0,016% voor `clear`.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1145004925&format=interactive",
  sheets_gid="1154751352"
  )
}}

In april 2020 heeft Google Chrome 81 de nieuwe Badging API verzonden, gevolgd door Microsoft Edge 84 in juli. Nadat Chrome de API had verzonden, schoten de gebruikscijfers omhoog. In oktober 2020 wordt bij 0,025% van alle pagina's die in Google Chrome worden geladen, de `setAppBadge()` - methode aangeroepen. De `clearAppBadge()` - methode wordt minder vaak aangeroepen, wanneer ongeveer 0,016% van de pagina's wordt geladen.

> ### <span lang="en">Notification Triggers API</span>

Reviewer: ```suggestion
### <span lang="en">Notification Triggers API</span> {notification-triggers-api}
```

  description="Diagram met Notification Triggers API-gebruik, gebaseerd op het percentage pagina's dat in Chrome wordt geladen met deze functie. Het vertoont een piek in maart 2020 met ongeveer 0,00003% van de paginaladingen, en daalt tot nul in oktober 2020.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1388597384&format=interactive",
  sheets_gid="1740370570"
  )
}}

Het Fugu-team experimenteerde eerst met Notification Triggers in een oorspronkelijke proef van Chrome 80 tot 83, waarbij de ontwikkeling later werd onderbroken vanwege het gebrek aan feedback van ontwikkelaars. Beginnend met Chrome 86 uitgebracht in oktober 2020, is de API weer in de oorspronkelijke proeffase terechtgekomen. Dit verklaart ook de gebruiksgegevens van de Notification Triggers API die piekten toen ze werden aangeroepen op 0,000032% van de pagina's die in Chrome werden geladen tijdens de eerste oorspronkelijke proef rond maart 2020.

> ## <span lang="en">Screen Wake Lock API</span>

Reviewer: ```suggestion
## <span lang="en">Screen Wake Lock API</span> {screen-wake-lock-api}
```

{{ figure_markup(
  image="idle_detection_api.png",
  alt="Percentage pagina's dat wordt geladen in Chrome met behulp van Idle Detection API",
  caption='Percentage pagina\'s dat wordt geladen in Chrome met behulp van Idle Detection API.<br>(Bron: <a hreflang="en" href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">Idle Detection</a>)',
  description="Grafiek van API-gebruik voor inactieve detectie, gebaseerd op het percentage pagina's dat met deze functie in Chrome wordt geladen. Er zijn alleen gegevens beschikbaar voor juli en oktober 2020, wat een zeer lage acceptatie van de API aantoont.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=963792757&format=interactive",
  sheets_gid="1324588405"
  )
}}

Op het moment van schrijven bevindt de Idle Detection API zich in een oorspronkelijke proef, dus de API-vorm kan in de toekomst veranderen. Om dezelfde reden is het gebruik ervan erg laag en nauwelijks meetbaar.

> ## <span hreflang="en">Periodic Background Sync API</span> {periodic-background-sync-api}

Reviewer: ```suggestion
## <span lang="en">Periodic Background Sync API</span> {periodic-background-sync-api}
```

  console.log(app.id, app.platform, app.url);
});
```

{{ figure_markup(
  image="get_installed_related_apps.png",
  caption="Aantal pagina's dat `getInstalledRelatedApps()` gebruikt.",
  description="Grafiek van het gebruik van `getInstalledRelatedApps()`, op basis van het aantal pagina's dat wordt gecontroleerd door HTTP Archive. Het vergelijkt het gebruik op mobiele en desktop-apparaten. Het laat een gestage groei zien voor mobiele apparaten, met een piek van 363 pagina's in oktober 2020 vergeleken met 44 desktoppagina's.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1774881171&format=interactive",
  sheets_gid="860146688",
  sql_file="get_installed_related_apps_usage.sql"
  )
}}

In de loop van 2020 vertoont de `getInstalledRelatedApps()` API een gestage groei op mobiele websites. In oktober maakten 363 mobiele pagina's die werden gevolgd door het HTTP Archive gebruik van deze API. Op desktoppagina's groeit de API niet zo snel. Dit kan ook het gevolg zijn van het feit dat Android-winkels momenteel aanzienlijk meer apps aanbieden dan de Microsoft Store voor Windows.

> ## <span lang="en">Content Indexing API</span>

Reviewer: ```suggestion
## <span lang="en">Content Indexing API</span> {context-indexing-api}
```
